graphistry/PlotterBase.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, *args, **kwargs):
 
 
     def __repr__(self):
-        bindings = ['edges', 'nodes', 'source', 'destination', 'node', 
+        bindings = ['edges', 'nodes', 'source', 'destination', 'relation', 'node', 
```

Comment:
can we call this edge_type ?

Reply:
within GNN side, we might want to create a different data rep than 1 big edge table + 1 big node table, e.g., type-indexed property tables, and the same for triple versions like <src.id, src.type, src.col1, ...., edge.id, edge.type, edge.col1 ... dst....> . Will be curious how that shapes up (scale concerns), for now we can hide from the user and do purpose-specific...